common/src/main/java/com/wynntils/sockets/model/HadesModel.java

```diff
@@ -47,20 +48,25 @@ public final class HadesModel extends Model {
 
     @Override
     public void init() {
-        tryCreateConnection();
+        if (Managers.WynntilsAccount.isLoggedIn()) {
+            tryCreateConnection();
+        }
     }
 
     @Override
     public void disable() {
         tryDisconnect();
     }
 
-    private void tryCreateConnection() {
```

Comment:
Most of the PR looks good, but this one kind of bugged me. If Athena login works but Hades connection fails, we will not get any feedback on this. Just like the Athena firstWorld message, maybe we need to do something similar for Hades?

Reply:
Fixed